.github/workflows/test.yml

```diff
@@ -138,6 +138,14 @@ jobs:
     runs-on: ubuntu-latest
     needs: build
     timeout-minutes: 60
+    strategy:
+      matrix:
+        git-provider: [github, gitlab]
+        include:
+        - git-provider: github
+          GIT_PROVIDER: github
```

Comment:
It seems `GIT_PROVIDER` is set in `actions/run-accepance-test` using the parameters `git-provider` in the previous line. Having this two variables with the same value makes me wonder  if we need it twice here for a particular reason?

Reply:
It was the only way I could make it work. I'm open to suggestions ...

Reply:
where is GIT_PROVIDER used ?

Reply:
in .github/actions/run-acceptance-test/action.yml

Reply:
do you have the error output when this is not set?

Reply:
No, this env only applies to the `smoke-test-long` job which is running tests in parallel. The only way I could tell it failed was when I manually checked if the parallel jobs were running on both gitlab and github repos